math/minuit2/src/MnPosDef.cxx

```diff
@@ -100,7 +101,9 @@ MinimumError MnPosDef::operator()(const MinimumError &e, const MnMachinePrecisio
 
    print.Warn("Matrix forced pos-def by adding to diagonal", pAdd);
 
-   return MinimumError(err, MinimumError::MnMadePosDef);
+   MinimumError epdf(err, MinimumError::MnMadePosDef);
+   std::cout << epdf.Hessian().size() << "  " << epdf.Hessian().Nrow() << std::endl;
```

Comment:
Do we need this printout that looks it was added for debugging?

It comes from the 4th commit. Maybe that commit should be amended?